modules/Microsoft.HealthcareApis/workspaces/iotconnectors/fhirdestinations/readme.md

```diff
@@ -0,0 +1,142 @@
+| `enableDefaultTelemetry` | bool | `True` |  | Enable telemetry via the Customer Usage Attribution ID (GUID). |
+| `location` | string | `[resourceGroup().location]` |  | Location for all resources. |
+| `resourceIdentityResolutionType` | string | `'Lookup'` | `[Create, Lookup]` | Determines how resource identity is resolved on the destination. |
+
+
+### Parameter Usage: `destinationMapping`
+
+You can specify a collection of destination mapping using the following format:
+
+> NOTE: More detailed information on destination mappings can be found [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/iot/how-to-use-fhir-mappings).
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"destinationMapping": {
+    "value": {
+        "templateType": "CodeValueFhir",
+        "template": {
+            "codes": [
+                {
+                    "code": "8867-4",
+                    "system": "http://loinc.org",
+                    "display": "Heart rate"
+                }
+            ],
+            "periodInterval": 60,
+            "typeName": "heartrate",
+            "value": {
+                "defaultPeriod": 5000,
+                "unit": "count/min",
+                "valueName": "hr",
+                "valueType": "SampledData"
+            }
+        }
+    }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+destinationMapping: {
+    templateType: 'CodeValueFhir'
+    template: {
+        codes: [
+            {
+                code: '8867-4'
+                system: 'http://loinc.org'
+                display: 'Heart rate'
+            }
+        ],
+        periodInterval: 60,
+        typeName: 'heartrate'
+        value: {
+            defaultPeriod: 5000
+            unit: 'count/min'
+            valueName: 'hr'
+            valueType: 'SampledData'
+        }
+    }
+}
+```
+
+</details>
+
+## Outputs
+
+| Output Name | Type | Description |
+| :-- | :-- | :-- |
+| `iotConnectorName` | string | The name of the medtech service. |
+| `location` | string | The location the resource was deployed into. |
+| `name` | string | The name of the FHIR destination. |
+| `resourceGroupName` | string | The resource group where the namespace is deployed. |
+| `resourceId` | string | The resource ID of the FHIR destination. |
+
+## Deployment examples
+
+<h3>Example 1: min</h3>
+
+<details>
+
+<summary>via Bicep module</summary>
+
+```bicep
+module iotConnector_destination './Microsoft.HealthcareApis/workspaces/iotconnectors/fhirdestinations/deploy.bicep' = {
+    name: '${uniqueString(deployment().name)}-test-iomtmap'
+    params: {
+        // Required parameters
+        name: '<<namePrefix>>iomtmap001'
+        workspaceName: '<workspaceName>'
+        iotConnectorName: '<eventHubName>'
+        destinationMapping '<destinationMapping>'
+        fhirServiceResourceId: '<fhirServiceResourceId>'
+    }
+}
+```
+
+</details>
+
+## Cross-referenced modules
+
+_None_
```

Comment:
As I cannot comment it anywhere else: Note that `modules/Microsoft.HealthcareApis/workspaces/iotconnectors/fhirdestinations/version.json` is empty